modules/charon-core/src/test/java/org/wso2/charon3/core/protocol/endpoints/GroupResourceManagerTest.java

```diff
@@ -747,12 +763,180 @@ public void testUpdateGroupWithPUTNotImplementedException(String id, String scim
+        };
+    }
+
+    @Test(dataProvider = "dataForUpdateWithPATCH")
+    public void testUpdateWithPATCH(String existingId, String patchRequest,
+                                    String attributes, String excludeAttributes, Object scimNewGroupObject,
+                                    Object scimOldGroupObject)
+            throws BadRequestException, CharonException, NotImplementedException, NotFoundException {
+
+        Group groupNew = (Group) scimNewGroupObject;
+        Group groupOld = (Group) scimOldGroupObject;
+        abstractResourceManager.when(()
+                -> AbstractResourceManager.getResourceEndpointURL(SCIMConstants.USER_ENDPOINT))
+                .thenReturn(SCIM2_GROUP_ENDPOINT);
+        Mockito.when(userManager.getGroup(anyString(), anyMap())).thenReturn(groupOld);
+        Mockito.when(userManager.patchGroup(anyString(), anyString(), anyMap(), anyMap())).thenReturn(groupNew);
+        SCIMResponse scimResponse = groupResourceManager.updateWithPATCH(existingId, patchRequest,
+                userManager, attributes, excludeAttributes);
+        Assert.assertEquals(scimResponse.getResponseStatus(), ResponseCodeConstants.CODE_OK);
+
+    }
+
+    @DataProvider(name = "dataForUpdateWithPATCHOverride")
+    public Object[][] dataToUpdateWithPATCHOverride()
+            throws BadRequestException, CharonException, InternalErrorException {
+
+        SCIMResourceTypeSchema schema = SCIMResourceSchemaManager.getInstance().getGroupResourceSchema();
+        JSONDecoder decoder = new JSONDecoder();
+        Group groupOld = decoder.decodeResource(NEW_GROUP_SCIM_OBJECT_STRING, schema, new Group());
+        String id = groupOld.getId();
+        Group groupNew = decoder.decodeResource(NEW_GROUP_SCIM_OBJECT_STRING_FOR_PATCH_UPDATE, schema, new Group());
+        return new Object[][]{
+                {id, SCIM2_PATCH_REQUEST_STRING, groupNew, groupNew}
+        };
+    }
+
+    @Test(dataProvider = "dataForUpdateWithPATCHOverride")
+    public void testUpdateWithPATCHOverride(String existingId, String patchRequest,
+                                            Object scimNewGroupObject, Object scimOldGroupObject)
+            throws BadRequestException, CharonException, NotImplementedException, NotFoundException {
+
+        Group groupNew = (Group) scimNewGroupObject;
+        Group groupOld = (Group) scimOldGroupObject;
+
+        abstractResourceManager.when(()
+                -> AbstractResourceManager.getResourceEndpointURL(SCIMConstants.USER_ENDPOINT))
+                .thenReturn(SCIM2_GROUP_ENDPOINT);
+        Mockito.when(userManager.getGroup(anyString(), anyMap())).thenReturn(groupOld);
+        Mockito.when(userManager.patchGroup(anyString(), anyString(), anyMap(), anyMap())).thenReturn(groupNew);
+        SCIMResponse scimResponse = groupResourceManager.updateWithPATCH(existingId, patchRequest,
+                userManager);
```

Comment:
merge this into 1 line

Reply:
Fixed in commit 34423db0111992d6450b72b5cb1f2b50ab84aec0